src/Nethermind/Nethermind.Runner/Ethereum/Steps/StartBlockProducerAuRa.cs

```diff
@@ -46,25 +46,26 @@ namespace Nethermind.Runner.Ethereum.Steps
     public class StartBlockProducerAuRa : StartBlockProducer
```

Comment:
would be great ti have it in AuRa assemblies if posssible

Reply:
StartBlockProducer and all base steps would have to go to main  Blockchain projects

Reply:
I see Otherwise Nethermind.Bootstrap project would maybe solve it? (by moving base steps there?)